resources/views/components/otp-input.blade.php

```diff
@@ -32,60 +32,83 @@
     :state-path="$getStatePath()"
 >
     <div x-data="{
-    	    state: $wire.$entangle('{{ $getStatePath() }}'),
-    	    length: {{$numberInput}},
-    	    autoFocus: '{{$isAutofocused}}',
-    	    type: '{{$inputType}}',
-            init: function(){
-                if (this.autoFocus){
-                    this.$refs[1].focus();
-                }
-            },
-            handleInput(e, i) {
-                const input = e.target;
-                if(input.value.length > 1){
-                    input.value = input.value.substring(0, 1);
+        state: $wire.$entangle('{{ $statePath }}'),
```

Comment:
I would separate style fixes and functional changes into separate PRs for easier reviews